.github/workflows/deploy-main.yml

```diff
@@ -48,10 +48,10 @@ jobs: # We need two separate builds as each is going to a different ECR repo
     needs: build-prod
     with:
       deploy_to_environment: prod
-      lambda: event-delivery
+      lambda: indexer
       docker_image_version: ${{ needs.build-prod.outputs.build_date }}
     secrets:
-      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID_PROD }}
       aws_role_id: ${{ secrets.AWS_ROLE_ID }}
-      aws_region: ${{ secrets.AWS_REGION }}
+      aws_region: us-west-2
```

Comment:
why not use variable?

Reply:
https://github.com/elastic-ipfs/event-delivery-lambda/pull/1#discussion_r930239640